src/vs/base/common/platform.ts

```diff
@@ -80,7 +85,7 @@ if (typeof navigator === 'object' && !isElectronRenderer) {
 	_isIOS = (_userAgent.indexOf('Macintosh') >= 0 || _userAgent.indexOf('iPad') >= 0 || _userAgent.indexOf('iPhone') >= 0) && !!navigator.maxTouchPoints && navigator.maxTouchPoints > 0;
 	_isLinux = _userAgent.indexOf('Linux') >= 0;
 	_isWeb = true;
-	_locale = navigator.language;
+	_locale = window.localStorage.getItem('vscode.nls.locale') || navigator.language;
```

Comment:
For some reason, this addition in this file causes [this integration test](https://github.com/microsoft/vscode/blob/b297fc96b45c26d9cf77e08421ef4f9f0ca99cf2/extensions/vscode-api-tests/src/singlefolder-tests/languages.test.ts#L99) to fail constantly and I'm not sure why...

Reply:
Perhaps I can get this value another way.. like through the AMD loader's configuration.